app/views/records/show_fields/_rights.html.erb

```diff
@@ -1,4 +1,4 @@
 <% record.rights.each do |r| %>
-  <%=link_to_field('rights',r, Sufia.config.cc_licenses_reverse[r])%> <%= iconify_auto_link(r,false) %><br />
+  <%=link_to_field('rights',r, Sufia.config.cc_licenses_reverse[r])%><% if is_url?(r) %> <%=iconify_auto_link(r,false) %><% end %><br />
```

Comment:
discussed with @mjgiarlo last week. i was of the opinion making the change here would have a more limited impact; changing it in iconify_auto_link would be changing the behavior of a public method. But if y'all believe that is the intended behavior of that method more broadly then we can certainly move it there.